### ISSUE TITLE
Update tab styles

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#1655]
The tabs component got a design overhaul.

**Design**: https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components-Post?type=design&node-id=70-18&mode=design&t=gh5btDmzaOgNux8c-0

**Optimization Ticket**: https://github.com/swisspost/design-system/issues/1125

**Scope**: These styles should only be applied to the tabs web component. The ngb-tabs should stay as they are to ensure backwards compatibility.

```[tasklist]
### Tasks
- [x] blocked by https://github.com/swisspost/design-system/issues/1654
- [ ] Color variants are no longer supported
- [ ] https://github.com/swisspost/design-system/issues/1938
```

**Comment by gfellerph** [comment]
Closing in favor of a future tab v2 ticket. Breaking changes are no longer being implemented for v1 components.
